playgrounds/vite-react/src/App.tsx

```diff
@@ -158,8 +160,9 @@ function SignMessage() {
       <h2>Sign Message</h2>
 
       <form
-        onSubmit={(event) => {
+        onSubmit={async (event) => {
           event.preventDefault()
+          await new Promise((resolve) => setTimeout(resolve, 1_000))
           const formData = new FormData(event.target as HTMLFormElement)
           signMessage({ message: formData.get('message') as string })
```

Comment:
`signMessage` does not redirect over to MM iOS app because 1s delay above cause universal link to expire.